packages/kbn-esql-validation-autocomplete/src/code_actions/actions.ts

```diff
@@ -223,7 +223,10 @@ async function getSpellingActionForFunctions(
   const errorText = queryString.substring(error.startColumn - 1, error.endColumn - 1);
   // fallback to the last command if not found
   const commandContext =
-    ast.find((command) => command.location.max > error.endColumn) || ast[ast.length - 1];
+    ast.find(
```

Comment:
Fixes the quick fixes (actions) bug https://github.com/elastic/kibana/issues/180155

Reply:
> and many others

I see lots of work here in the quick fixes code, but only one new test case. Are there more fixes we can validate in our tests?

Is there anything I should test manually beyond https://github.com/elastic/kibana/issues/180155?